agent-bootstrap/agent-bootstrap.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
   id "com.github.johnrengelman.shadow"
 }
 
-apply from: "${rootDir}/gradle/java.gradle"
+apply from: "$rootDir/gradle/java.gradle"
```

Comment:
And it also causes merge conflicts with ongoing work 😭, but I'm not opposed to it as a general cleanup. It might be easier on these merge PRs to review commit-by-commit, then you can skip over commits like this that are just cosmetic.

Reply:
I even not sure this can be classified as "useful cleanup" :)

Reply:
sorry for the noise.